src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Samples.cshtml

```diff
@@ -0,0 +1,292 @@
+@page "{id?}/{revisionId?}"
+@model APIViewWeb.Pages.Assemblies.UsageSamplePageModel
+@{
+    Layout = "ReviewLayout";
+    ViewData["Title"] = "Usage Samples";
+}
+<link rel="stylesheet" href="~/usagesample.css">
+
+@if (Model.Sample.SampleId == "Bad Deployment") // If the samples container does not exist
+{
+    <div class="container-fluid">
+        <div class="mx-5 mt-2">
+            <div class="text-muted">
+                <p> <strong> Usage Samples are not available for this APIView deployment. </strong></p>
+                <p> Please check the new DOCUMENTING.md to see how to enable them. </p>
+            </div>
+        </div>
+    </div>
+}
+else if (Model.Sample.SampleId == "File Content Missing") // if the file is broken in some way
+{
+    <div class="container-fluid">
+        <div class="mx-5 mt-2">
+            <div class="text-muted">
+                <p class="text-danger"> <strong> Bad blob read.</strong></p>
```

Comment:
Might be too technical of a header. Can we say something more user facing, e.g. "Failed to load usage sample for this review"?